src/utils/hooks/transaction/utils/fee.ts

```diff
@@ -152,6 +152,19 @@ const getActionAmount = (
       break;
     }
     /* END - AMM */
+    /* START - LOANS */
+    case Transaction.LOANS_REPAY:
+    case Transaction.LOANS_LEND: {
+      const [, amount] = params.args;
+      amounts = [amount];
+      break;
+    }
+    case Transaction.LOANS_REPAY_ALL: {
+      const [, calculatedLimit] = params.args;
+      amounts = [calculatedLimit];
+      break;
+    }
+    /* END - LOANS */
```

Comment:
What about withdrawing? In case when user wants to withdraw maximum of tokens, but it would be possible to pay for tx fee in `qToken` we will have to deduct tx fee amount from maximum withdrawable amount. I would just add a comment for this here as this scenario is not very possible but better to have it noted down.

Reply:
withdrawing does not affect negatively your balance, but instead adds to your balance.

Reply:
It affects your balance of `qToken`, you send qToken and get underlying currency for it.